.github/workflows/release-please.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  # test:
+  #   uses: ./.github/workflows/test.yml
+
+  release-please:
+    runs-on: ubuntu-latest
```

Comment:
Please freeze to a specific ubuntu version, similar to https://github.com/ButterCMS/buttercms-ruby/pull/28/files#diff-2c84033033d49186c63e6adcd705f63b11ae6814cd76c152c9c486d389fbccf3R23